magicctapipe/scripts/lst1_magic/lst1_magic_mc_dl0_to_dl1.py

```diff
@@ -352,6 +373,7 @@ def mc_dl0_to_dl1(input_file, output_dir, config, focal_length):
                     n_pixels=n_pixels,
                     n_islands=n_islands,
                     magic_stereo=magic_stereo,
+                    tels_with_trigger=float(tels_trigger),
```

Comment:
saving it as a float is a bad idea. It already fails with 6 telescopes (this is in ipython, which seem to save still many digits for a float, but saving this to the file I think it will fail even faster):
```
In [7]: a="1000200030004005006"                                                 

In [8]: float(a)                                                                
Out[8]: 1.000200030004005e+18

In [9]: int(float(a))                                                           
Out[9]: 1000200030004004992
```

one could save it as int, but I'm afraid that this can also run out of int size when writing to file. Why not simply save it as a string separating the tel_ids with a coma?

Reply:
When trying to save it as a string it complained about this and did not save it. But I can try again

Reply:
Ok, if we use a string, the stereo step crashes:

```
Reconstructing the stereo parameters...
0 events
10 events
Traceback (most recent call last):
  File "/home/evisentin/mambaforge/envs/ctap19/bin/lst1_magic_stereo_reco", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/home/evisentin/mambaforge/envs/ctap19/lib/python3.11/site-packages/magicctapipe/scripts/lst1_magic/lst1_magic_stereo_reco.py", line 441, in main
    stereo_reconstruction(args.input_file, args.output_dir, config, args.magic_only)
  File "/home/evisentin/mambaforge/envs/ctap19/lib/python3.11/site-packages/magicctapipe/scripts/lst1_magic/lst1_magic_stereo_reco.py", line 370, in stereo_reconstruction
    save_pandas_data_in_table(
  File "/home/evisentin/mambaforge/envs/ctap19/lib/python3.11/site-packages/magicctapipe/io/io.py", line 1382, in save_pandas_data_in_table
    f_out.create_table(group_name, table_name, createparents=True, obj=data_array)
  File "/home/evisentin/mambaforge/envs/ctap19/lib/python3.11/site-packages/tables/file.py", line 1002, in create_table
    descr, _ = descr_from_dtype(obj.dtype, ptparams=self.params)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/evisentin/mambaforge/envs/ctap19/lib/python3.11/site-packages/tables/description.py", line 846, in descr_from_dtype
    raise NotImplementedError(
NotImplementedError: structured arrays with columns with type description ``object`` are not supported yet, sorry
```
If we try to write it as a list, it won't be written.
Any idea? Do we really need this parameter?

Reply:
what about writing it as an int which refer to each possible telescope combination (i.e., as the combo_type)?

Reply:
strings are problematic because there is no fixed size. In principle they can be written out only if a maximum number of characters is specified. But the solution with the binary encoding may be suitable (i.e. the same as writing out a int value).